mockup-contributions_list.php

```diff
@@ -0,0 +1,922 @@
+<?php
```

Comment:
I was thinking of that. Should we maybe just have people do inline css for now (On top of the page)? until we come up with a better solution?